packages/combobox/package.json

```diff
@@ -31,7 +31,7 @@
     "@leafygreen-ui/popover": "^11.0.2",
     "@leafygreen-ui/tokens": "^1.4.1",
     "@leafygreen-ui/typography": "^15.1.1",
-    "chalk": "^5.1.2",
+    "chalk": "^4.1.2",
```

Comment:
https://github.com/chalk/chalk/releases/tag/v5.0.0

I could not get v5 to work with Jest so I downgraded to v4. The version was updated from v2 to v5 about a [week](https://github.com/mongodb/leafygreen-ui/commit/b7f7a4c9531450988f74216bfef90b2ea82ef2ec#diff-99d5cfad21705571b5979762c25112410a72f6134b232fbe565023e188c8d4a9R34) ago.